packages/playwright-core/src/client/page.ts

```diff
@@ -564,10 +564,26 @@ export class Page extends ChannelOwner<channels.PageChannel> implements api.Page
     return this.mainFrame().locator(selector, options);
   }
 
+  get(selector: string, options?: LocatorOptions): Locator {
+    return this.mainFrame().locator(selector, options);
```

Comment:
consider aligning all functions to use `this.mainFrame()` instead of `this._mainFrame` or vise-versa